src/users.spec.ts

```diff
@@ -194,10 +192,8 @@ describe('Users...', () => {
       // Got one
       ctx = ctx.withThreadName('foo')
       const id = ThreadID.fromRandom()
-      // Update existing db config directly as it doesn't yet directly support Context API on method calls
-      db.config = ctx
-      // @todo: In the near future, this should be `await db.newDB(id, ctx)`
-      await db.newDB(id.toBytes())
+      
+      await db.newDB(id, ctx)
```

Comment:
```suggestion
      await db.newDB(id, ctx.withThreadName('foo'))
```

That changed, coupled with the deletion above feels cleaner to me, and is now possible :)